packages/stone-phaser/PKGBUILD

```diff
@@ -0,0 +1,41 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Christopher Arndt <aur -at- chrisarndt -dot- de>
+
+pkgname=stone-phaser
+pkgver=0.1.2
+pkgrel=1
+pkgdesc='A classic analog phaser effect LADSPA, LV2 and VST2 plugin'
+arch=(x86_64 aarch64)
+url='https://github.com/jpcima/stone-phaser'
+license=(Boost)
+depends=(cairo glibc gcc-libs)
```

Comment:
`glibc` and `gcc-libs` are both pulled in by the `base` package, i.e. part of every Arch installation by default.
The Arch wiki about this:
> and may or may not be listed, for example [glibc](https://archlinux.org/packages/?name=glibc) cannot be uninstalled as every system needs some C library

Since there are no clear rules we could probably decide on our own rules, what do you think?

Reply:
I put these in, since the non-GUI libs do not need cairo, but glibc and gcc-libs are a special case, as you mention.

From observation, he practice in the official repose leans towards including these.

Maybe @dvzrv can give some input?

Reply:
I see. I'm fine with either solution :)

Reply:
I always add glibc and gcc-libs to not have transitive dependencies (although every system should have the base package installed, which pulls these two in).